cmd/talosctl/cmd/mgmt/gen/secureboot.go

```diff
@@ -120,10 +119,8 @@ func generateSigningCerts(path, prefix, commonName string, rsaBits int, outputCe
 			return err
 		}
 
-		if outputDER {
```

Comment:
is it unused now?

Reply:
Yes, it is unnecessary. It kinda made sense when we were outputting both the cert and the signing public key, but not anymore. It was never bound to a flag.

Reply:
ah, makes sense... I thought it's a flag.